specification/configuration.md

```diff
@@ -130,6 +136,34 @@ beyond the OpenTelemetry specification exist.
   - Distribution MUST default to `otlp`
   - Distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
 
+#### Real User Monitoring Libraries
+
+Real User Monitoring (RUM) instrumentation libraries cannot use environment variables for configuration.
+Instead, they MUST expose a `SplunkRum` object/class/namespace (depending on the language used) that allows
+setting the properties listed below (in a language-specific way: Java may use builders, Swift can use named
+parameters with default values, JavaScript can use objects, etc.).
+
+RUM instrumentation libraries MUST support the following configuration properties:
+
+| Property (default value)               | Description                                                                                                                                                                                                 |
+| -------------------------------------- | -----------                                                                                                                                                                                                 |
+| `realm` ()                             | Splunk realm, e.g. `us0`, `us1`. If set, value of `beaconEndpoint` will be automatically computed based on this. [1] [2] [3]                                                                                |
+| `beaconEndpoint` ()                    | RUM beacon URL, ex. `https://rum-ingest.<realm>.signalfx.com/v1/rum`. If both `realm` and `beaconEndpoint` are set, `beaconEndpoint` takes precedence. [1] [2] [3]                                          |
```

Comment:
I suggest `e.g.` instead of `ex.` 

AFAIK `ex.` often means "excluding" and usually means "exercise" ;)